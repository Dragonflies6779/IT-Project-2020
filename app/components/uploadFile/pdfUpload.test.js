var React = require('react');
import renderer from 'react-test-renderer';
import { render, fireEvent } from '@testing-library/react';
import { screen ,getByLabelText,
  getByText,
  getByTestId,
  queryByTestId,
  toBeDisabled} from '@testing-library/dom'
var Upload = require('./pdfUpload.js');

it('renders correctly', () => {
  const tree = renderer.create(<Upload/>).toJSON();
  expect(tree).toMatchSnapshot();
});

it('The button is not disabled and works', () => {
  const utils = render(<Upload />)
  const input = utils.getAllByRole('button')
  expect(input[0].hasAttribute('disabled')).toBe(false)
  expect(input[1].hasAttribute('disabled')).toBe(false)
})
