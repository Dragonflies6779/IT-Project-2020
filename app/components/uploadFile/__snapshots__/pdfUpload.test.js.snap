// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="card-profile"
>
  <input
    onChange={[Function]}
    type="file"
  />
  <button
    onClick={[Function]}
  >
    Upload Resume
  </button>
  <button
    onClick={[Function]}
  >
    Download Resume
  </button>
  <p>
    
  </p>
  <p>
    
  </p>
  <p />
</div>
`;
